docs/stack/get-started/install/docker.md

```diff
@@ -81,12 +81,18 @@ To pass in arbitrary configuration changes, you can set any of these environment
 
 * `REDISBLOOM_ARGS`: arguments for RedisBloom
 
+
 For example, here's how to use the `REDIS_ARGS` environment variable to pass the `requirepass` directive to Redis:
 
 {{< highlight bash >}}
 docker run -e REDIS_ARGS="--requirepass redis-stack" redis/redis-stack:latest
 {{< / highlight >}}
 
+An example of setting [Redis persistence](https://redis.io/docs/management/persistence/):
+{{< highlight bash >}}
+docker run -e REDIS_ARGS="--save 60 1000" --appendonly yes" redis/redis-stack:latest
```

Comment:
The quoting appears to be incorrect. There's a first set of quotes around the REDIS_ARGS variable, the second is dangling.